gtk/transmission-gtk.metainfo.xml.in

```diff
@@ -12,7 +12,7 @@ Copyright 2017 Endless Mobile, Inc.
   <summary>Download and share files over BitTorrent</summary>
 
   <description>
-    <!-- Translators: these are the application description paragraphs in the AppData file. -->
+    <!-- Translators: these are the application description paragraphs in the MetaInfo file. -->
```

Comment:
You need to update `po/POTFILES.in` as well.

Reply:
thanks you!